### ISSUE TITLE
AOD screen customization

### ISSUE THREAD

**Issue by rdnt** [rdnt/m8#6]
- ~Allow showing complications on AOD~ would cause increased battery consumption, and also the ambient font size is bigger, so less room for complications
- Allow using the default block font for AOD

**Comment by rdnt** [comment]
Addressed on v2.6.4

**Comment by Jnss98b** [comment]
> also the ambient font size is bigger, so less room for complications

Maybe you can limit showing complications on the AOD only when the big AOD clock setting is disabled? All of the first party pixel watch faces show complications on AOD so I miss having them at a glance when using m8 sometimes.

**Comment by rdnt** [comment]
Good point. I will re-open the issue, but this needs some further thought on how to do right, in order to easily integrate with #14.

**Comment by LuminarySage** [comment]
I see this is implemented with the "Detailed" AOD option. Is it also possible to have the clock be outlined at the same time as well?

**Comment by rdnt** [comment]
Here are some modified screenshots with how the outline and bold outline styles would look in the sport layout, for example.

![image](https://github.com/rdnt/m8/assets/17600197/83e4bfa8-3d02-414e-a5c2-0e47eb3cafbb)
![image](https://github.com/rdnt/m8/assets/17600197/3575fca1-18a2-403a-9bfa-af041f54b34c)

I think they look a little bit out of place (the reason I changed my mind about implementing this in the first place), but since it's needed, I'm considering adding this as part of #23.
